source/vir/src/well_formed.rs

```diff
@@ -152,17 +152,16 @@ fn check_path_and_get_function<'a>(
                 ));
             } else {
                 let path = path_as_friendly_rust_name(&x.path);
-                return Err(error(
-                    span,
-                    &format!(
-                        "`{path:}` is not supported (note: you may be able to add a Verus specification to this function with `assume_specification`){:}",
-                        if x.path.is_rust_std_path() {
-                            " (note: the vstd library provides some specification for the Rust std library, but it is currently limited)"
-                        } else {
-                            ""
-                        },
-                    ),
-                ));
+                let mut error = error(span, &format!("`{path:}` is not supported."));
+                let mut help = "You may be able to add a Verus specification to this function with `assume_specification`.".to_string();
+                if x.path.is_rust_std_path() {
+                    help += " The vstd library provides some specification for the Rust std library, but it is currently limited.";
+                }
+                if *x.path.last_segment() == "deref" {
+                    help += " For dereferncing in ghost mode, you should explicitly call `.deref()` other than using `*`. This restriction may be lifted in the future.";
```

Comment:
small typo with "dereferncing".

Also, I'm not sure what this help message is for? Using explicit `.deref()` and using `*` to implicity insert a `.deref()` ought to be equivalent, regardless of whether we're in ghost code or not.